catlin/pkg/cmd/validate/validate.go

```diff
@@ -26,12 +27,40 @@ import (
 	"github.com/tektoncd/plumbing/catlin/pkg/validator"
 )
 
+func getFilesInDir(path string) ([]os.FileInfo, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	files, err := f.Readdir(-1)
+	if err != nil {
+		return nil, err
+	}
+	return files, nil
+}
+
 func fileExists(path string) bool {
 	info, err := os.Stat(path)
 	if os.IsNotExist(err) {
 		return false
 	}
-	return !info.IsDir()
+
+	if info.IsDir() {
+
+		files, err := getFilesInDir(path)
+		if err != nil {
+			return false
+		}
+
+		for _, file := range files {
+			if filepath.Ext(file.Name()) == ".yaml" {
```

Comment:
Since we have the directory name as the filename shouldn't we just check for the name of the file with the extension of yaml ?? wdyt